html/index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<head>
+    <title>My Personal Website</title>
+    <style></style>
+</head>
+<body>
+    <header>
+        <img src="/media/cc0-images/grapefruit-slice-332-332.jpg">
+        <div class = "quicknav">
+            <a href="#News">News</a>
+            <a href="#contact">Contact</a>
+            <a href="#about">About</a>
+          </div>
```

Comment:
```suggestion
        <nav class="quicknav">
            <a href="#News">News</a>
            <a href="#contact">Contact</a>
            <a href="#about">About</a>
       </nav>
```
This part is the navigation of your app and the `nav` element should be used here instead of a `div` element. [See MDN.](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav).

And you need to remove the spaces of `class = "quicknav"` for this attribute to work.

And I would suggest you to not use capital letters for your links as it is easier to handle. ⚓